dojo/api_v2/serializers.py

```diff
@@ -561,21 +557,6 @@ def save(self):
         skipped_hashcodes = []
         try:
             for item in parser.items:
-                if skip_duplicates:
```

Comment:
@Maffooch a little confused about this removal. These hash codes aren't related to dedupe. Would you please explain?

Reply:
These hash codes were computed in APIv2 as a way to preform a different kind of deduplication than that of the GUI and APIv1. They were deleted so that all three methods of Dojo interaction could be consistent.

Reply:
I'm using this so not in favor of removing this.